containers-toolkit/Public/ContainerdTools.psm1

```diff
@@ -16,6 +16,10 @@ function Get-ContainerdLatestVersion {
 }
 
 function Install-Containerd {
+    [CmdletBinding(
+        SupportsShouldProcess,
+        ConfirmImpact = 'High'
```

Comment:
q: does it mean you will need to confirm for each tool or there's a way to pass-through the confirmation if you are installing the whole bundle? This will be helpful especially for un-attended setups.

Reply:
Yes. But using $Confirm:$False bypasses the prompt. See the [PR description](https://github.com/microsoft/containers-toolkit/pull/6#issue-2192392230)